README.mdx

```diff
@@ -0,0 +1,198 @@
+# TODO: Expand on outline
+
+> :warning: This guidance is in prep for a Playground tutorial, but attachments must first be enabled on Playground for
+this guidance to make sense.
+
+# KittyVerse + KittyHats
+
+To showcase how attachments unlock permissionless composability on Flow, this tutorial will walk you though making your
+own cat-inspired NFT collection and a wardrobe of hats to go along with them. Let's put some hats on some cats!
+
+## What is composability?
+
+Before digging in, let's talk about what we mean by **_composability_**.
+
+<Callout type="info">
+
+Composability is the ability for existing components in one construction to be reused and even extended for another 
+purpose. <br/>
+
+Think of composable architectures like [building with legos](https://twitter.com/cdixon/status/1450462593232089088?s=20) 
+- you build something you need as a block or set of blocks and someone else can come along and use those same blocks to 
+build something entirely new.
+
+Composability on Flow, especially with attachments is permissionless because not only can you build with existing 
+components, but no one can stop you from doing so!
+
+</Callout>
+
+Permissionless composability is huge because, as builders, we can build on existing constructs for existing 
+communities and be sure that the original developer won't rug our access - AKA platform risk.
+
+For an example of the negative case, just look at how big tech companies locked down their API in the early days.
+
+Less duplicate code, easier to bootstrap attention and adoption, and users benefit from the utility you built into 
+something they already presumably own and love - win-win-win!
```

Comment:
```suggestion
Permissionless composability is one of the unique characteristics of decentralized app development because, as builders, we can freely compose richer or derivative functionalities on top of any existing contracts from other applications or communities. Contracts that you deploy which interact with other contracts are not subject to internal changes or control by the contract you composed it upon. 

Less duplicate code, easier to bootstrap attention and adoption, and users benefit from the utility you built into 
something they already presumably own and love - win-win-win!
```

I struggle a little with the 'original developer won't rug our access' since for Web3 newcomers it might not be enough for them to make a connection. We may want to drop the point about platform risk - it's still the case that if you build to Flow you can't easily switch platforms whereas the same doesn't apply to EVM compatible equivalents. 

I'm also not sure there's enough strength in the reference to big tech APIs for the suggested wording. GCP copied AWS in terms of APIs to smooth migration, so in a sense the locked down aspects of the cloud provider APIs is their strength since they are so ubiquitously supported/similar

Reply:
> it's still the case that if you build to Flow you can't easily switch platforms whereas the same doesn't apply to EVM compatible equivalents.

Definitely a valid point. By "platform" here I mean something like Twitter's API when they proclaimed to be a protocol, not necessarily the the blockchain itself. The ground may at any point fall out from under you if the chain ceases to exist, but, taking the chain's existence as a given, I'd say the immutability guarantees of smart contracts provide a risk-reduced platform in comparison to privately owned and managed APIs. Case in point being the aforementioned Twitter API.

Regardless, your verbiage gets the point across more clearly for sure. Thanks for the rephrase!